CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ endif()
 
 # Custom preprocessor flags
 if(DEFINED CMAKE_MAXIMUM_RANK)
-  set(fyppFlags "-DMAXRANK=${CMAKE_MAXIMUM_RANK}")
+  set(fyppFlags "-DMAXRANK=${CMAKE_MAXIMUM_RANK} -I${PROJECT_SOURCE_DIR}/include")
 elseif(f03rank)
   set(fyppFlags)
```

Comment:
This is the wrong place to add this option. Please append the option later or initialize the `fyppFlags` with the include directory and append the `MAXRANK` macro here.